### PR TITLE
README updated

### PR DESCRIPTION
**Pull request by sarastrasner**
- [X] Top-level README “Table of Contents” is updated
 - [ ] Feature tasks for this challenge are completed
 - [ ] Unit tests written and passing
     - [ ] “Happy Path” - Expected outcome
     - [ ] Expected failure
     - [ ] Edge Case (if applicable/obvious)
 - [X] README for this challenge is complete
     - [X] Summary, Description, Approach & Efficiency, Solution
     - [ ] Link to code
     - [X] Picture of whiteboard

### CODE REVIEW

File: javascript/linkedList/linkedList/README.md [hunk 1]
```diff
@@ -70,4 +70,16 @@ Write a method for the Linked List class which takes a number, k, as a parameter
...
+Zip two linked lists.
+
+## Challenge Description
+Write a function called zipLists which takes two linked lists as arguments. Zip the two linked lists together into one so that the nodes alternate between the two lists and return a reference to the head of the zipped list. Try and keep additional space down to O(1). You have access to the Node class and all the properties on the Linked List class as well as the methods created in previous challenges.
+
+## Approach & Efficiency
+My partner and I whiteboarded for an hour, and didn't get as far ad we'd hoped to. We were contemplatng how to integrate some of the other methods into our solution and ran out of time. 
+
+## Solution
+![LL k-th value whiteboard](./assets/LL-zip.png)
```
Comment (adrianhuebner) [inline javascript/linkedList/linkedList/README.md:85]: Really great looking whiteboard! I would suggest changing the alt text to the LL zip or something similar!
Would like to see code and tests added!
